packages/backend/src/card/service.ts

```diff
@@ -0,0 +1,54 @@
+import { AxiosInstance } from 'axios'
+import { Logger } from 'pino'
+
+const PAYMENT_FOUNDED_PATH = '/payment-event'
+
+type EventDetails = {
+  requestId: string
+  outgoingPaymentId: string
+  resultCode: string
+}
+
+export interface CardService {
+  sendPaymentEvent(eventDetails: EventDetails): Promise<void>
+}
+
+interface ServiceDependencies {
+  axios: AxiosInstance
+  cardServiceHost: string
+  logger: Logger
+}
+
+export async function createCardService(
+  deps_: ServiceDependencies
+): Promise<CardService> {
+  const logger = deps_.logger.child({
+    service: 'card-service'
```

Comment:
nit: use PascalCase to match other child logger aliases

`card-service` -> `CardService`